packages/arcs/src/index.ts

```diff
@@ -0,0 +1,13 @@
+export * from './arc_labels'
+export * from './arc_link_labels'
```

Comment:
```suggestion
export * from './arc-labels'
export * from './arc-link-labels'
```

If you decide to switch over from underscore to dashes, make this change as well.